symphony/bdk/core/service/user/user_service.py

```diff
@@ -796,6 +796,40 @@ async def list_audit_trail(
             params['after'] = after
         return await self._audit_trail_api.v1_audittrail_privilegeduser_get(**params)
 
+    @retry
+    async def list_all_audit_trail(
+            self,
+            start_timestamp: int,
+            end_timestamp: int = None,
+            initiator_id: int = None,
+            role: RoleId = None,
+            chunk_size: int = 100,
+            max_number: int = None
+    ) -> AsyncGenerator[V1AuditTrailInitiatorList, None]:
+        """Returns an asynchronous generation of audit trail of actions performed by a privileged user in a given period
+        of time.
+        See: `List Audit Trail v1 <https://developers.symphony.com/restapi/reference#list-audit-trail-v1>`_
```

Comment:
Please change the URL to:
`https://developers.symphony.com/restapi/reference/list-audit-trail-v1`